src/ResourceManager/Resources/Commands.Resources/Models.Authorization/AuthorizationClientExtensions.cs

```diff
@@ -117,6 +117,7 @@ private static IEnumerable<PSRoleAssignment> ToPSRoleAssignments(this IEnumerabl
 
             List<string> objectIds = new List<string>();
             objectIds.AddRange(assignments.Select(r => r.PrincipalId.ToString()));
+            objectIds = objectIds.Distinct().ToList();
```

Comment:
@darshanhs90 would you mind adding test coverage for this change?

Reply:
Done